R/read.any.R

```diff
@@ -204,34 +207,68 @@ read.any <- function(
   }
 
   if(!is.null(widths) && is.null(col.names) || tolower(file_type) == 'fwf'){
-    x = read.fwf(filename, widths, header = FALSE)
+    x = utils::read.fwf(filename, widths, header = FALSE)
   }
 
   if(!is.null(widths) && !is.null(col.names) || tolower(file_type) == 'fwf'){
-    x = read.fwf(filename, widths, header = FALSE, col.names = col.names)
-  }  
- 
+    x = utils::read.fwf(filename, widths, header = FALSE, col.names = col.names)
+  }
+    
+  if(grepl("\\.pbix$", filename)) {
+      
+    # Validate pbi-tools is downloaded.
+    if (shell('pbi-tools info') != 0) {
+      message("pbi-tools is not configured correctly")
```

Comment:
Please include the URL https://pbi.tools/ in the message and any other important info to help the user get it set up.